EnglishFilter/sections/foreign.txt

```diff
@@ -4702,6 +4702,8 @@ wroclawskiejedzenie.pl,innpoland.pl,smartage.pl,joemonster.org,natemat.pl,mamadu
 trojmiasto.pl#%#AG_abortInlineScript(/banner_/, '$');
 ! filman.cc - skip timer
 filman.cc#%#(function(){var c=window.setTimeout;window.setTimeout=function(f,g){return g===1e3&&/#kam-ban-player/.test(f.toString())&&(g*=0.01),c.apply(this,arguments)}.bind(window)})();
+! https://github.com/AdguardTeam/AdguardFilters/issues/42437
+inpost.pl##.app-screen
```

Comment:
It looks like some kind of self-promo, so could you please move this rule to `AnnoyancesFilter - self-promo.txt`?